### ISSUE TITLE
Error installing due to prereq failing

### ISSUE THREAD

**Issue by OkGoDoIt** [Microsoft/vs-tools-for-ai#36]
Tried installing today after updating VS to latest 15.8.2, but the extension says it is missing a prerequisite.  In digging into the logs I found it's failing due to Microsoft.VisualStudio.Component.Common.Azure.Tools:

```
8/29/2018 4:05:48 AM - The extension with ID 'ec62d75b-40eb-4fad-806c-c8196b2bd091' is not installed to Microsoft Visual Studio Enterprise 2017.
8/29/2018 4:05:50 AM - Extension cannot be installed to the following products due to missing prerequisites:
8/29/2018 4:05:50 AM - 	Microsoft Visual Studio Enterprise 2017
8/29/2018 4:05:50 AM - 		-------------------------------------------------------
8/29/2018 4:05:50 AM - 		Identifier   : Microsoft.VisualStudio.Component.Common.Azure.Tools
8/29/2018 4:05:50 AM - 		Name         : Connectivity and publishing tools
8/29/2018 4:05:50 AM - 		Version      : [1.10,2.0)
8/29/2018 4:05:50 AM - 		Error        : The prerequisite version specified does not match the version installed
8/29/2018 4:05:50 AM - 
8/29/2018 4:05:50 AM - System.InvalidOperationException: No modifications could be applied.
```

I'm not entirely sure how to check the version I have installed, but both the VS installer and the VS extension manager say I have everything fully up to date, including everything Azure-related.

**Comment by linmajia** [comment]
@OkGoDoIt , thank you very much for the feedback.

VS 15.8.2 update is released just now and changed the version definition of Microsoft.VisualStudio.Component.Common.Azure.Tools. We will fix this issue ASAP.

Thank you very much!

**Comment by linmajia** [comment]
@OkGoDoIt , this issue is fixed in the new version 0.5.0.752675 . Please go to menu **Tools** > **Extensions and Updates** and install / update VS Tools for AI.

Thank you very much.

**Comment by OkGoDoIt** [comment]
Yes, this appears to be working now.  Thanks!
